### DISCUSSION TITLE
External links and `typedRoutes: true` – what's the best way to handle them?

### DISCUSSION THREAD

**Discussion by philwolstenholme** [vercel/next.js#46511] [Help] [answered]
Hi,

I've just turned on `typedRoutes: true`, really like it so far.

I was wondering what people are doing about external links? They error at the moment as there's no corresponding type for them. I'm doing this sort of thing at the moment:

```jsx
{/* @ts-expect-error: External link upsets Next typedRoutes */}
<Link href="https://wolstenhol.me" className={cx(linkClasses)}>
  External link
</Link>
```

If there's a nicer way of handling it, or if this approach is OK, then should we add this to [the docs](https://beta.nextjs.org/docs/configuring/typescript#statically-typed-links)?

**Comment by sroussey** [comment, marked as answer]
Why not use a normal a link? I thought the purpose of the Link tag was to do special NextJS stuff, and it can not do it for external links, so why bother?

**Reply by philwolstenholme** [reply]
Oh my gosh, of course! 🤦‍♂️ I'd just used `<Link>` without thinking, and ignored poor old `<a>`!

I think that's the answer and I'll swap that external link in my codebase to use a good old fashioned anchor element!
